setup.py

```diff
@@ -109,7 +109,7 @@ def read(filename):
 
 
 dependencies = [
-    'click>=2.0.0',
+    'click>=7.0.0',
```

Comment:
Should this include a max version i.e. `click>=7.0.0,<8` to prevent surprise API breaks in the future?

Reply:
Can’t tell in advance if the tests will be affected by new breaking changes.
Don’t know if we want to limit ourselves to a specific version at this time.
If v8 comes out and updating is not worth the effort, then setting a max makes sense.